src/core/data/apartment.ts

```diff
@@ -0,0 +1,82 @@
+import {
```

Comment:
omg, you have to wrap this into another one "abstraction". Looks like code generation.

Reply:
[similar answer](https://github.com/akveo/kittenTricks/pull/98#discussion_r276565419)